### ISSUE TITLE
PDFUA-179: Figure with space as Alt text | FAIL

### ISSUE THREAD

**Issue by zakkinsey** [zakkinsey/PDFA-TEST#176]
Jira issue originally created by user @:

This example demonstrates the failure condition that applies for an image that conveys information, but the relevant information is not conveyed by the [alt text](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#alt-text). In this case there is only a space set as [alt text](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#alt-text).
 
Please note: A simple machine check will not return an error message related to missing [alt text](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#alt-text) here.

**Comment by zakkinsey** [comment]
Comment created by @MarkusErle:

Cleaned up with Roman's tool
